data/nav.yml

```diff
@@ -84,6 +84,8 @@
           path: "agent/v3/prioritization"
         - name: "Securing"
           path: "agent/v3/securing"
```

Comment:
Note for later - worth thinking about where we want to link to the signed pipelines docs from elsewhere. I imagine it'd make sense to mention on "Securing" at a minimum

Reply:
@moskyb just reminding you of this one ^^

Reply:
i've added it to the file below: https://github.com/buildkite/docs/pull/2524/files#diff-b27d77ac575fad5917c6b6719e5898132289acc351e73db4dd27987dda9aef4e

Reply:
Ah, apologies - missed it. Resolving :)